pkg/autodiscovery/providers/file.go

```diff
@@ -305,5 +307,8 @@ func GetIntegrationConfigFromFile(name, fpath string) (integration.Config, error
 		return config, errors.New("the 'docker_images' section is deprecated, please use 'ad_identifiers' instead")
 	}
 
+	// Interpolate env vars. Returns an error a variable wasn't subsituted, ignore it.
+	_ = configresolver.SubstituteTemplateEnvVars(&config)
```

Comment:
Should we not support env vars in all providers ?

Reply:
All the other providers already support env vars because they support templates, which are a superset of env vars.